manager/manager.go

```diff
@@ -2,6 +2,7 @@ package manager
 
 import (
 	"fmt"
+	"github.com/hashicorp/go-multierror"
```

Comment:
nit: can you `goimports` on this file?

Reply:
Re-ordered the imports 👍  (if that's what you meant ^^ )